src/content/guides/tooltips.md

```diff
@@ -352,6 +352,61 @@ class App extends React.Component {
 ReactDOM.render(<App/>, mountNode);
 ```
 
+## Victory Native
+In Victory Native tooltips are presented when a data element is pressed and hidden when the press is released. A `VictoryVoronoiContainer` is required for the chart to enable the press events to show tooltips. On press it will show the user the tooltip for the closest data point. Since data points can be quite small showing the closest data point increases the tap targets for the tooltip. Set `VictoryVoronoiContainer` as the `container` prop to the chart itself, like `VictoryScatter`, or the `VictoryChart` wrapper.
+
+```playground
+<VictoryChart 
+  containerComponent={<VictoryVoronoiContainer />}
+>
+  <VictoryScatter
+    labelComponent={<VictoryTooltip />}
```

Comment:
This is an oversight carried over from the `victory-native` demos, but please add `style={{ data: { fill: (d) => d.fill } }}` to pick up the fills on these points. (We used to pick style props off `data` automatically, but it was a source of issues)